internal/buildengine/filehash.go

```diff
@@ -63,18 +66,17 @@ func CompareFileHashes(oldFiles, newFiles FileHashes) (FileChangeType, string, b
 
 // ComputeFileHashes computes the SHA256 hash of all (non-git-ignored) files in
 // the given directory.
-func ComputeFileHashes(module Module) (FileHashes, error) {
-	config := module.Config
-
+func computeFileHashes(config moduleconfig.ModuleConfig, patterns []string) (FileHashes, error) {
```

Comment:
We don't seem to be using anything but config.Dir in here, can we just pass that in instead of the entire ModuleConfig?